install.sh

```diff
@@ -32,22 +32,39 @@ fi
 
 # Test if Git is available on the Host
 if ! hash git 2>/dev/null; then
-  >&2 echo "Error: Unable to find git in your PATH."
+  echo >&2 "Error: Unable to find git in your PATH."
   exit 1
 fi
 
 # Clone the Shorebird repository into the install_dir
 echo "Cloning Shorebird into $(install_dir)"
 git clone https://github.com/shorebirdtech/shorebird.git -b stable "$(install_dir)"
 
+# Build Shorebird
+eval $(install_dir)/bin/shorebird --version
```

Comment:
Why is the eval needed instead of a direct call?

Reply:
Because it hasn't been sourced yet (users need to either restart their terminal or manually export the path.

Reply:
Confused.  Does `shorebird` depend on `shorebird` being in path?  Why would it matter if the path had been updated?  even if it does one could set the path just for that command with "PATH=/path/to/bin:$PATH /path/to/shorebird". No objection to eval though. 🤷

Reply:
Yes `shorebird` does depend on `shorebird` being in the path. Yeah we could set the path just for that command I'm fine either way.